Nautilus/Assets/Gadgets/EquipmentGadget.cs

```diff
@@ -20,6 +20,7 @@ public class EquipmentGadget : Gadget
     public QuickSlotType QuickSlotType { get; set; }
 
 
+
```

Comment:
These spaces are for readability, I think that spacing a little some parts is sometimes better for readability